tree/tree/src/TTree.cxx

```diff
@@ -4839,7 +4861,7 @@ Int_t TTree::FlushBaskets() const
    Int_t nb = lb->GetEntriesFast();
 
 #ifdef R__USE_IMT
-   if (ROOT::IsImplicitMTEnabled() && fIMTEnabled) {
```

Comment:
One should imply the other - no way to set `fIMTEnabled` if `IsImplicitMTEnabled` is false - right?